### DISCUSSION TITLE
How to build facebook folly library with rules_foreign_cc?

### DISCUSSION THREAD

**Discussion by fuhailin** [bazelbuild/rules_foreign_cc#616] [Q&A]
Anyone could place an example with `folly`?

**Comment by UebelAndre** [comment]
From slack

> Folly is a standard CMake build system so it should work with rules_foreign_cc with the cmake rule. The biggest challenge is going to be getting all the required dependencies also built.
For what its worth looks like there are native bazel build files around for folly too: https://github.com/smyte/smyte-db/blob/master/third_party/folly.BUILD

**Comment by UebelAndre** [comment]
Based on that message, sounds like you need to define additional targets for the dependencies?
